ros2controlcli/ros2controlcli/verb/load_start_controller.py

```diff
@@ -31,8 +30,8 @@ def add_arguments(self, parser, cli_name):
         add_controller_mgr_parsers(parser)
 
     def main(self, *, args):
+        print("deprecated warning: Please use either 'load --set_state' or 'set_state'")
         response = load_start_controller(args.controller_manager, args.controller_name)
         if not response.ok:
-            print('Error loading and starting controller, check '
-                  'controller_manager logs', file=sys.stderr)
-        return not response.ok
+            return 'Error loading and starting controller, check controller_manager logs'
+        return 'Successfully loaded and started controller {}'.format(args.controller_name)
```

Comment:
```suggestion
        return f'Successfully loaded and started controller {args.controller_name}'
```

let's be better than printf :D format strings are supported starting from python3.6, we are well within that range with 3.8.5 on ubuntu20

Reply:
https://github.com/ros-controls/ros2_control/issues/355